lib/aur-build

```diff
@@ -385,7 +395,8 @@ while IFS= read "${read_args[@]}" -ru "$fd" path; do
 
     if (( create_package )); then
         if (( chroot )); then
-            PKGDEST="$var_tmp" aur chroot "${chroot_args[@]}" --build "${chroot_build_args[@]}"
+            PKGDEST="$var_tmp" AUR_CHROOT_UID=$(id -u) \
+                auth_cmd aur "${AUR_BUILD_CHROOT:-chroot}" "${chroot_args[@]}" --build "${chroot_build_args[@]}" -- "${makepkg_common_args[@]}" "${makepkg_args[@]}"
```

Comment:
See L129; the additional arguments after `--` will be interpreted by the existing `aur--chroot` as arguments for `base_packages`.